### PR TITLE
Edited topics

### PR DESCRIPTION
**Pull request by philip-menzies**
docs\get_started\plan_deployment\docker\index.md
docs\get_started\plan_deployment\hybrid_deployment\index.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_creating_site\rm_setup_content_author\rm_workflow_access\rm_create_site_admin_workflow_author.md docs\get_started\plan_deployment\traditional_deployment\roadmaps\rm_creating_site\rm_setup_content_author\rm_workflow_access\rm_create_site_admin_workflow_reviewer.md docs\get_started\software_support.md

### CODE REVIEW

File: docs/get_started/plan_deployment/hybrid_deployment/index.md [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
 
 ## Overview
 
-The HCL Digital Experience 9.5 Hybrid deployment reference architecture and topics describe an approach to deploy and manage HCL Digital Experience 9.5 core Portal Server and Web Content Manager services on premises, and connect to cloud native components Digital Asset Management, Content Composer, Experience API and related services using Helm, in a production environment.
+The HCL Digital Experience 9.5 Hybrid deployment reference architecture and topics describe how to deploy and manage HCL Digital Experience 9.5 core Portal Server and Web Content Manager services on premises. The resources also explain how to connect to cloud native components, Digital Asset Management, Content Composer, Experience API and related services by using Helm, in a production environment.
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/hybrid_deployment/index.md:7]:
```suggestion
The HCL Digital Experience 9.5 Hybrid deployment reference architecture and topics describe how to deploy and manage HCL Digital Experience 9.5 core Portal Server and Web Content Manager services on premises. The resources also explain how to connect to cloud native components, Digital Asset Management, Content Composer, Experience API and related services by using Helm in a production environment.
```

Comment (philip-menzies) [reply docs/get_started/plan_deployment/hybrid_deployment/index.md:7]: Yes!

File: docs/get_started/plan_deployment/hybrid_deployment/index.md [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
--   Practitioner Studio is enabled in the Digital Experience 9.5 CF198 or higher installation. See the [How to enable Practitioner Studio](../../../build_sites/practitioner_studio/working_with_ps/enable_prac_studio.md) topic for instructions.
--   A common domain that uses an SSL connection is established for both the on-premise HCL DX 9.5 CF198 and higher on-premise environments and the target Red Hat Open Shift or Amazon EKS, Azure AKS, or Google GKE platform deployment to contain the cloud native components (HCL DX Experience API, Digital Asset Management and Content Composer).
+-   HCL Digital Experience V9.5 CF198 or a later release is deployed to supported on-premises platforms in a standalone, cluster, or farm topology. See [Roadmaps to deploy your Digital Experience 9.5 system](../traditional_deployment/roadmaps/index.md) for more information.
+-   Practitioner Studio is enabled in the Digital Experience 9.5 CF198 or a installation. See [How to enable Practitioner Studio](../../../build_sites/practitioner_studio/working_with_ps/enable_prac_studio.md) for instructions.
```
[the remark this answers is not in the document]
Comment (philip-menzies) [reply docs/get_started/plan_deployment/hybrid_deployment/index.md:15]: Yes!

File: docs/get_started/plan_deployment/hybrid_deployment/index.md [hunk 3]
```diff
@@ -1,33 +1,32 @@
...
-
-    -   **Step 3: Proceed to configuration instructions** listed in this Help Center topic: [Prepare configuration](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configuration.md).
+1. Configure the network for the on-premises DX 9.5 CF198 or later deployment so that the HCL DX 9.5 Container Update CF198 or later components are accessible externally from the Kubernetes or OpenShift platform cluster. See [Configure Networking](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configure_networking.md)  in the Helm Planning sections of the Help Center.
+2. Set the Core application deployment parameter in your ``custom-values.yaml`` file to `false`. See [Planning your container deployment using Helm](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configuration.md#custom-value-files) in the Help Center for more information.
+3. Proceed by following configuration instructions** listed in  [Prepare configuration](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configuration.md).
```
Comment (biancaluna) [inline docs/get_started/plan_deployment/hybrid_deployment/index.md:31]:
```suggestion
3. Proceed by following configuration instructions listed in  [Prepare configuration](../../../deployment/install/container/helm_deployment/preparation/mandatory_tasks/prepare_configuration.md).
```

Comment (philip-menzies) [reply docs/get_started/plan_deployment/hybrid_deployment/index.md:31]: Yes

File: docs/get_started/software_support.md [hunk 4]
```diff
@@ -1,28 +1,24 @@
-# Program Requirements
```
Comment (biancaluna) [inline docs/get_started/software_support.md:1]: Hi @philip-menzies , we would have to revert the changes here because this is a document from legal.

Comment (philip-menzies) [reply docs/get_started/software_support.md:1]: No problem. Leave the language that legal provided!

**Review by biancaluna — APPROVED** [review]
